pkg/controller/plan/adapter/ocp/validator.go

```diff
@@ -21,7 +21,7 @@ type Validator struct {
 	plan         *api.Plan
 	inventory    web.Client
 	sourceClient k8sclient.Client
-	log          logging.LevelLogger
```

Comment:
can't we replace it with `logr.Logger`? then it should be the only change

Reply:
you're right, changed